From d63290a3e9ee9c9e02bcc96ececb63e7c782893f Mon Sep 17 00:00:00 2001
From: Philippe Gervais <philippe.n.gervais@free.fr>
Date: Tue, 2 May 2023 10:11:42 +0200
Subject: [PATCH 1890/1936] Updated NEWS.txt before a release

Also adjusted TODO-release.md to take into account the actual
configurations that can be compiled.
---
 NEWS.txt        |  7 +++++++
 TODO-release.md | 10 ++++++----
 2 files changed, 13 insertions(+), 4 deletions(-)

diff --git a/NEWS.txt b/NEWS.txt
index 8ce5a73a..4c8cef2c 100644
--- a/NEWS.txt
+++ b/NEWS.txt
@@ -3,11 +3,18 @@
         - Add portamento auto mode
         - Add MIDI bank select to LV2/VST2 versions
         - Add filter bypass
+        - Add sympathetic resonance effect
         - Fix built in record wavefile saving
         - Fix message corruption bug in rtosc
         - Fix analog filter q update rate
         - Fix all-note-off message with sustain pedals
+        - Fix how controller values are applied when enabled/disabled
         - Increase parameter smoothing filter frequency
+        - Eliminate some clicks when changing filter parameters
+        - Fix loading invalid values from XML savefiles
+        - Fix and improve the Zyn-Fusion UI
+        - Fix multiple bugs
+
 
 3.0.6 (21 January 2022)
         - Enhances watchpoint system for waveform views (GSOC michiboo)
diff --git a/TODO-release.md b/TODO-release.md
index b13c2bf7..abafaf6c 100644
--- a/TODO-release.md
+++ b/TODO-release.md
@@ -11,13 +11,15 @@
 
 ## Build and test a release candidate
 
-All applies to the 4 UI options: no-gui, fltk, ntk and zyn-fusion.
-
-- Check the code builds on Linux, Mac and Windows (standalone and plugin versions)
+- Check the code builds on Linux: no gui, fltk, ntk, zest.
+  Standalone and plugin versions.
+- Check the code builds on Windows: zest. Standalone and plugin versions.
+- Check the code builds on Mac: fltk, standalone only.
 - Check the docker containers produce binaries
 - Check the code runs on Linux, Mac and Windows (standalone and plugin versions)
 
-Then tag the release git commit with the version number.
+Then tag the release git commit with the version number (zynaddsubfx,
+mruby-zest-build).
 
 ## Publish the new version
 
-- 
2.47.0

